_includes/events/schedule.html

```diff
@@ -60,7 +60,7 @@ <h2 class="mb-20 unbold text-center">Thursday, July 27th</h2>
           <li>
             5:30 pm - Team pitches, team selection, initial team discussion
           </li>
-          <li>6:00 pm - Dinner with your team (and others!)</li>
+          <li>6:00 pm - Dinner with your team (and others!) - Southside Dining Hall</li>
```

Comment:
6 or 5:45?

Reply:
I could change it to 5:45pm if we're ok shortening the Team pithes, team selection, initial team discussion.